src/conventional/ionosonde_ascii2ioda.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import time
+from datetime import datetime
+import numpy as np
+import logging
+
+# These modules need the path to lib-python modules
+import pyiodaconv.ioda_conv_engines as iconv
+from pyiodaconv.orddicts import DefaultOrderedDict
+
+os.environ["TZ"] = "UTC"
+
+# these are the unique values in the raw input file
+varDict = {'criticalFrequency': ['criticalFrequency', "float", 'm s-1'],
+           'criticalFrequencyConfidence': ['criticalFrequencyConfidence', "float", 'fractional percent'],
+           'height': ['height', "float", "m"],
+           'electronDensity': ['electronDensity', "float", 'number / m^3'],
+           'electronDensityConfidence': ['electronDensityConfidence', "float", 'number / m^3'],
+           'ionosphericLayer': ['ionosphericLayer', "integer", 'IonosphericLayer']}              # F2   - this is index for layer (E=0; F1=1; F2=2)
+
+# these are the MetaData common to each input
+locationKeyList = [("latitude", "float", "degrees_north"),
+                   ("longitude", "float", "degrees_east"),
+                   ("dateTime", "long", "seconds since 1970-01-01T00:00:00Z"),
+                   ("hmF2", "float", "height of the F2 peak electron density"),
```

Comment:
Are these ("hmF2" and others) standard variable names? Otherwise, they are not self-explainable as we have had for the neutral atmosphere naming convention. This can be ignored for now but we have to discuss about naming convention soon